test/integration/template_admin_test.go

```diff
@@ -365,3 +366,36 @@ func TestGlobalLoadBalancerConfigRendersOnlyOnEntryPointCluster(t *testing.T) {
 		assert.NotContains(t, obj.Annotations, "nuodb.com/load-balancer-policy.zone1")
 	}
 }
+
+func TestAnnotationsRender(t *testing.T) {
+	// Path to the helm chart we will test
+	helmChartPath := testlib.ADMIN_HELM_CHART_PATH
+
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"admin.podAnnotations.key1": "value1",
+			"admin.podAnnotations.key2": "value2",
+			"admin.podAnnotations.key3\\.key3": "value3",
+			"admin.podAnnotations.key4\\.key4/key4": "value4",
+			"admin.podAnnotations.key5\\.key5/key5": "value5/value5",
+			"admin.podAnnotations.vault\\.hashicorp\\.com/agent-inject": `"true"`,
```

Comment:
Some time ago I noticed a problem when using `toYaml` with bool values represented as strings (https://github.com/helm/helm/issues/4262). I don't know if this breaks somehow Vault in case it is boolean `true` instead of the string representation.

Reply:
I tested generating with both string and bool which both generate fine.  However I did find an issue using bool in the tests and had to switch to string for it to pass.